lua/lvim/core/dap.lua

```diff
@@ -28,9 +28,11 @@ end
 M.setup = function()
   local dap = require "dap"
 
-  vim.fn.sign_define("DapBreakpoint", lvim.builtin.dap.breakpoint)
-  vim.fn.sign_define("DapBreakpointRejected", lvim.builtin.dap.breakpoint_rejected)
-  vim.fn.sign_define("DapStopped", lvim.builtin.dap.stopped)
+  if lvim.use_icons then
+    vim.fn.sign_define("DapBreakpoint", lvim.builtin.dap.breakpoint)
+    vim.fn.sign_define("DapBreakpointRejected", lvim.builtin.dap.breakpoint_rejected)
+    vim.fn.sign_define("DapStopped", lvim.builtin.dap.stopped)
+  end
```

Comment:
```suggestion
```
I don't think this is necessary, as it's better for them to just modify sign itself or overwrite it completely

edit: they're already set to ascii characters anyway https://github.com/mfussenegger/nvim-dap/blob/d6d8317ce9e096029150bc5844916347a9af6f45/lua/dap.lua#L131-L135

Reply:
so i should keep it?